Apps/Src/ReadCarCAN.c

```diff
@@ -189,21 +125,25 @@ void Task_ReadCarCAN(void *p_arg)
     );
     assertOSError(OS_READ_CAN_LOC, err);
 
-    //Start CAN Watchdog timer
+    // Start CAN Watchdog timer
     OSTmrStart(&canWatchTimer, &err);
     assertOSError(OS_READ_CAN_LOC, err);
-    
 
-    while (1)
-    {
+    while(1){
+
+        // Array Contactor is turned off if Ignition 1 is off
+        if(Minion_Read_Pin(IGN_1, &Merr) == OFF){
+            Contactors_Set(ARRAY_CONTACTOR, OFF, true);
+            assertOSError(OS_MINIONS_LOC, err);
```

Comment:
@KnockbackNemo @IshDeshpa Got it. 
I had assumed that I need to use an assertOSError after every minion pin read because the main loop in the Task_Init had one after reading the minion pin. Is the one in Task_Init intended to be there?

Reply:
assertOSError is only for when we run an OS function and want to make sure that it doesn't return an error. If there's any OS function being run, most (if not all) of the time it should have an assertOSError after it.